core/controller/src/main/scala/whisk/core/loadBalancer/LoadBalancerToKafka.scala

```diff
@@ -102,21 +99,9 @@ trait LoadBalancerToKafka extends Logging {
     def getIssueCountByInvoker(): Map[Int, Int] = invokerActivationCounter.readOnlySnapshot.mapValues(_.cur).toMap
 
     /**
-     * Convert user activation counters into a map of JsObjects to be written into consul kv
-     *
-     * @param userActivationCounter the counters for each user's activations
-     * @returns a map where the key represents the final nested key structure for consul and a JsObject
-     *     containing the activation counts for each user
+     * Retrieve a snapshot of activation counts issued per subject by load balancer
      */
-    protected def getUserActivationCounts(): Map[String, JsObject] = {
-        userActivationCounter.toMap mapValues {
-            _.cur
-        } groupBy {
-            case (key, _) => LoadBalancerKeys.userActivationCountKey + "/" + key.substring(0, 1)
-        } mapValues { map =>
-            map.toJson.asJsObject
-        }
-    }
+    def getUserActivationCounts(): Map[String, Long] = userActivationCounter.toMap mapValues { _.cur.toLong }
```

Comment:
Can we omit the `()` as this operation is no longer "side-effecting"?

Also: Why `Long` over `Int`? `Counter` only emits an `Int` (but in general I agree that we should use `Long` for these, but then everywhere, starting at `Counter`)